### PR TITLE
[REVIEW] Pin `dask` and `distributed` for release

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR pins `dask` and `distributed` to `2022.9.2` for `22.10` release.

xref: https://github.com/rapidsai/cudf/pull/11822

### CODE REVIEW

**Review by raydouglass — APPROVED** [review]

**Comment by rlratzel** [comment]
rerun tests

reason: CI problems should be fixed after merging #2684

File: conda/environments/cugraph_dev_cuda11.2.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ dependencies:
 - raft-dask=22.10.*
 - pylibraft=22.10.*
 - cuda-python>=11.5,<11.7.1
-- dask>=2022.7.1
-- distributed>=2022.7.1
+- dask==2022.9.1
+- distributed==2022.9.1
```
Comment (galipremsagar) [inline conda/environments/cugraph_dev_cuda11.2.yml:22]:
```suggestion
- dask==2022.9.2
- distributed==2022.9.2
```

File: conda/environments/cugraph_dev_cuda11.4.yml [hunk 2]
```diff
@@ -18,8 +18,8 @@ dependencies:
 - raft-dask=22.10.*
 - pylibraft=22.10.*
 - cuda-python>=11.5,<11.7.1
-- dask>=2022.7.1
-- distributed>=2022.7.1
+- dask==2022.9.1
+- distributed==2022.9.1
```
Comment (galipremsagar) [inline conda/environments/cugraph_dev_cuda11.4.yml:22]:
```suggestion
- dask==2022.9.2
- distributed==2022.9.2
```

File: conda/environments/cugraph_dev_cuda11.5.yml [hunk 3]
```diff
@@ -18,8 +18,8 @@ dependencies:
 - raft-dask=22.10.*
 - pylibraft=22.10.*
 - cuda-python>=11.5,<11.7.1
-- dask>=2022.7.1
-- distributed>=2022.7.1
+- dask==2022.9.1
+- distributed==2022.9.1
```
Comment (galipremsagar) [inline conda/environments/cugraph_dev_cuda11.5.yml:22]:
```suggestion
- dask==2022.9.2
- distributed==2022.9.2
```

File: conda/recipes/cugraph/meta.yaml [hunk 4]
```diff
@@ -57,8 +57,8 @@ requirements:
     - cudf={{ minor_version }}
     - dask-cudf {{ minor_version }}
     - dask-cuda {{ minor_version }}
-    - dask>=2022.7.1
-    - distributed>=2022.7.1
+    - dask==2022.9.1
+    - distributed==2022.9.1
```
Comment (galipremsagar) [inline conda/recipes/cugraph/meta.yaml:61]:
```suggestion
    - dask==2022.9.2
    - distributed==2022.9.2
```

**Comment by galipremsagar** [comment]
rerun tests

**Review by rlratzel — APPROVED** [review]

**Comment by rlratzel** [comment]
@gpucibot merge

**Pull request merged**